lib/client-assets.php

```diff
@@ -190,7 +190,8 @@ function wc_admin_print_script_settings() {
 			'userLocale'    => get_user_locale(),
 			'weekdaysShort' => array_values( $wp_locale->weekday_abbrev ),
 		),
-		'currentUserData'       => $current_user_data,
+		'currentUserData'  => $current_user_data,
+		'alertCount'       => WC_Admin_Notes::get_notes_count(),
```

Comment:
This needs to get restricted to just the count of `error` and `update` type notes - otherwise the placeholder can display even though no note will be shown.

Reply:
@jeffstieler I mentioned that in the PR. I updated the method with support for filtering by note type: https://github.com/woocommerce/wc-admin/pull/1758/commits/7e381f4e351a145d02756bb3edede9b434b428c6